src/reports/components/report-sections/collapsible-result-section.tsx

```diff
@@ -3,7 +3,7 @@
 import { NamedSFC } from 'common/react/named-sfc';
 import * as React from 'react';
 
-import { ResultSectionTitleV2 } from '../../../DetailsView/components/cards/result-section-title-v2';
+import { ResultSectionTitle } from '../../../DetailsView/components/cards/result-section-title';
```

Comment:
I think we can remove the .. chain after Sebastian's #982 (here and in two other places below).